mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -679,6 +683,10 @@ class ExperimentView extends Component {
       ];
       const paramsMap = ExperimentViewUtil.toParamsMap(paramsList[index]);
       const metricsMap = ExperimentViewUtil.toMetricsMap(metricsList[index]);
+
+      //add run name to export row
+      Utils.getRunName(tagsList[index]);
```

Comment:
Oops, sorry my last comment was unclear - I think using the logic from your last comment makes sense, i.e. we should replace `runInfo.name` on line 678 with `Utils.getRunName(...)`:

```
    const row = [
        runInfo.run_uuid,
        Utils.getRunName(tagsList[index]), // add run name to export row
        runInfo.source_type,
        runInfo.source_name,
        runInfo.user_id,
        runInfo.status,
      ];
```

It's a great point that `runInfo.name` should correctly be set to the value from the tag - to fix this we should actually modify the backend storage implementations (`mlflow/store/file_store.py` and `mlflow/store/sqlalchemy_store.py`) to read the run name from tags. Currently the REST API actually reads run name from the wrong location, hence why `runInfo.name` is set incorrectly. IMO that's actually a separate issue that's definitely worth fixing - in the short term I think the `Utils.getRunName(tagsList[index])` fix should address the CSV downloading bug

Reply:
@smurching replaced `runInfo.name` with the call to `Utils.getRunName(tagsList[index])`

Screenshot of csv export after the change:
<img width="752" alt="screen shot 2019-02-04 at 19 09 14" src="https://user-images.githubusercontent.com/1425269/52245631-6ed02280-28b0-11e9-830f-e505ab29fca4.png">